.yarn/versions/d075ed96.yml

```diff
@@ -0,0 +1,4 @@
+undecided:
```

Comment:
I have no clue what to do here. I'd say maybe that `@yarnpkg/core` should get a minor bump because we added the hook? And the others could be `patch`?

Reply:
Yep for the core 👍 On top of it, `@yarnpkg/cli` should also be minor because it's the source of truth for the bundled CLI we release.

Other packages can be set to "declined", since they don't mandate this new feature.

Reply:
Done